### PR TITLE
Ported RDS & S3 automatic setup script to main

### LINKED ISSUE #101: Port the ability to automatically setup RDS & S3 from v1.3 branch to main branch
**Is your feature request related to a problem? Please describe.**
Add the ability to automatically setup RDS & S3 from main branch

**Describe the solution you'd like**
Setup script and doc should be ported to main branch.

### PR DESCRIPTION
**Pull request by AlexandreBrown**
**Which issue is resolved by this Pull Request:**
Resolves #101 
  
- [x] end-to-end test from new cluster (main branch + manifest v1.4.1)

### CODE REVIEW

File: docs/deployment/rds-s3/README.md [hunk 1]
```diff
@@ -53,7 +53,38 @@ eksctl create cluster \
...
+The script also handles cases where the resources already exist in which case it will simply skips the step.
+  
+Note : The script will **not** delete any resource therefore if a secret already exist or if a database with the same name already exist or if the S3 bucket already exists for instance, it will skip those steps. This was done in order to prevent unwanted results such as accidental deletion. For instance, if a database with the same name already exists, the script will skip the database creation setup. If it's expected in your scenario, then perhaps this is fine for you, if you simply forgot to change the database name used for creation then this gives you the chance to retry the script with the proper value. See `python auto-rds-s3-setup.py --help` for the list of parameters as well as their default values.
+
+1. Install the dependencies for the script
+    1. Install the python dependencies `pip install -r requirements.txt`
+    2. Install [eksctl](https://docs.aws.amazon.com/eks/latest/userguide/eksctl.html#installing-eksctl)
```
Comment (surajkota) [inline docs/deployment/rds-s3/README.md:72]: 2 & 3 are already in the prerequisites in section 1.0, shall we remove those from here?

**Review by surajkota — APPROVED** [review]
@akartsky I have taken a quick look and planning to merge this PR since these changes have already been reviewed in #78 . Do you see any issues?

This will allow upcoming changes like #108 #98 etc. to keep the script up to date

File: docs/deployment/rds-s3/README.md (hunk 1 above)
Comment (AlexandreBrown) [reply docs/deployment/rds-s3/README.md]: Good idea @surajkota , I applied your suggestion and removed 2 & 3

File: docs/deployment/rds-s3/auto-rds-s3-setup.py [hunk 2]
```diff
@@ -0,0 +1,707 @@
...
+    print("Setting up Kubeflow Pipeline...")
+
+    print("Retrieving DB instance info...")
+    try:
+        db_instance_info = get_db_instance_info()
+    except get_rds_client().exceptions.DBInstanceNotFoundFault:
+        print("Could not retrieve DB instance info, aborting Kubeflow pipeline setup!")
+        return
+
+    pipeline_params_env_file = "../../../../apps/pipeline/upstream/env/aws/params.env"
```
Comment (AlexandreBrown) [inline docs/deployment/rds-s3/auto-rds-s3-setup.py:531]: This file path is wrong

Comment (AlexandreBrown) [reply docs/deployment/rds-s3/auto-rds-s3-setup.py]: Done

**Review by goswamig — APPROVED** [review]

**Comment by surajkota** [comment]
@AlexandreBrown please rebase from latest main

**Comment by AlexandreBrown** [comment]
@surajkota rebased from latest main

**Review by akartsky — APPROVED** [review]

**Pull request merged**